### PR TITLE
Feature/drive

### PR DESCRIPTION
**Pull request by elizaBarnett**
beginning of drive code.

### CODE REVIEW

File: src/main/java/frc/robot/Drive.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+package frc.robot;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+/**
+ * Add your docs here.
+ */
+public class Drive {
+
+    private Gearbox leftGearbox;
```
Comment (kushagharahi) [inline src/main/java/frc/robot/Drive.java]:
```suggestion
    private Gearbox rightGearbox;
```

**Comment by kushagharahi** [comment]
@elizaBarnett, really great job. This looks awesome. 

Lets figure out a folder structure for these classes and then get it merged. 

I think the reason only one side of the bot worked is because there are issues with a bunch of the motor controllers. We had to go through 5-7 of them just to find two working ones. Same issue with the former practice bot.

**Review by kushagharahi — DISMISSED** [review]
Good work. Lets figure out a folder structure and then get this merged into master to you can move on to a different task.

**Review by ghost — APPROVED** [review]
Approved

**Review by kushagharahi — APPROVED** [review]

**Pull request merged**